src/spatio_temporal_voxel_layer.cpp

```diff
@@ -1112,6 +1138,26 @@ void SpatioTemporalVoxelLayer::clearArea(
   CostmapLayer::clearArea(start_x, start_y, end_x, end_y, invert_area);
 }
 
+void SpatioTemporalVoxelLayer::clearCostmapLayerAroundPose(
+  double pose_x, double pose_y, double reset_distance)
+{
+  double start_point_x = pose_x - reset_distance / 2;
+  double start_point_y = pose_y - reset_distance / 2;
+  double end_point_x = start_point_x + reset_distance;
+  double end_point_y = start_point_y + reset_distance;
```

Comment:
Why is reset distance not divided by two here? It seems that the end points will be further from the start than the start points.

Reply:
Explained IRL. This works as it should